bin/crew

```diff
@@ -642,11 +642,13 @@ def download
       # Stow file in cache if requested
       if CREW_CACHE_ENABLED
         puts 'Caching downloaded archive...'
-          Dir.chdir "#{@extract_dir}" do
-            system "tar c#{@verbose}Jf #{CREW_CACHE_DIR}/#{filename}.tar.xz \
-              $(find -mindepth 1 -maxdepth 1 -printf '%P\n')"
-          end
-        system "sha256sum #{CREW_CACHE_DIR}/#{filename}.tar.xz > #{CREW_CACHE_DIR}/#{filename}.tar.xz.sha256"
+        Dir.chdir @extract_dir do
```

Comment:
Not that it ever should happen but what if there are spaces in the directory path?

Reply:
I don't even think homebrew supports spaces in directory paths.

Reply:
Just because homebrew doesn't support it doesn't mean we need to follow suit.  We're not copycats.

Reply:
It is fine since URL will convert spaces to `%20` string